src/edge_proxy/settings.py

```diff
@@ -136,6 +136,9 @@ class AppSettings(BaseModel):
 
 
 class AppConfig(AppSettings, BaseSettings):
+    class Meta:
+        extra = "ignore"
```

Comment:
This is already the default per https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.extra

Reply:
Hmm... yes, but it's overridden specifically in the BaseSettings class as per [this](https://github.com/pydantic/pydantic-settings/blob/main/pydantic_settings/main.py#L405). 

That being said, I expected (based on this) that something like `docker run --rm -e FOO_BAR=foo flagsmith/edge-proxy:2.17.0` would also generate the error, but it didn't. 

Probably more investigation is needed, although the solution I added [here](https://github.com/Flagsmith/edge-proxy/pull/141/commits/92a2142cbd5d8344d5b5181f2aab7a25797e06a9) does prevent the error reported in the issue when using `API_POLL_FREQUENCY`, but perhaps it also has some other knock on effects that need investigating before (if) we merge this.

Reply:
I have tested the behaviour and verified that the code here does resolve the issue - it prevents the exception as well as behaving as expected if you to set `API_POLL_FREQUENCY` environment variable (or any other aliased name of a given setting). 

I think we should go ahead and merge this.